### PR TITLE
feat: Adding more info to vehicle journeys

### PR DESCRIPTION
**Pull request by alainajaffer**
The RDS should be updated to store the following TXC values:

VehicleJourney / DepartureTime 

VehicleJourney / VehicleJourneyCode (already there)

VehicleJourney / Operational / TicketMachine / JourneyCode

I have also added the function to get the journeys for a given service

### CODE REVIEW

**Review by mohit-kpmg — DISMISSED** [review]

File: packages/api/client.ts [hunk 1]
```diff
@@ -371,6 +371,87 @@ export const getServices = async (dbClient: Kysely<Database>, input: ServicesQue
...
+            "vehicle_journeys.departureTime",
+            "services.destination",
+            "services.origin",
+            "service_journey_patterns.direction",
+        ])
+        .distinct()
+        .where("services.id", "=", service.id)
+        .where("dataSource", "=", input.dataSource)
+        .where("vehicle_journeys.departureTime", "is not", null)
+        .$narrowType<{ departureTime: NotNull }>()
```
Comment (DeannaSharma) [inline packages/api/client.ts]:
```suggestion
```

As discussed, let's remove this from the query and create a formatJourneys function to remove any results that have a null departure time to speed up the query

File: packages/api/get-service-journeys.ts [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+    const adminAreaCodes = pathParameters?.adminAreaCodes ?? "";
+    const adminAreaCodeArray = adminAreaCodes
+        .split(",")
+        .filter((adminAreaCode) => adminAreaCode)
+        .map((adminAreaCode) => adminAreaCode.trim());
+
+    if (adminAreaCodeArray.length > Number(MAX_ADMIN_AREA_CODES)) {
+        throw new ClientError(`Only up to ${MAX_ADMIN_AREA_CODES} administrative area codes can be provided`);
+    }
```
Comment (DeannaSharma) [inline packages/api/get-service-journeys.ts]: Let's remove references to adminAreaCodes in this file and in the db query as I don't think it's necessary for the consequence page

File: packages/api/client.ts (hunk 1 above)
Comment (alainajaffer) [reply packages/api/client.ts]: done

File: packages/api/get-service-journeys.ts (hunk 2 above)
Comment (alainajaffer) [reply packages/api/get-service-journeys.ts]: done

**Review by DeannaSharma — APPROVED** [review]

**Pull request merged**